task/init/0.2/migrations/0.2.4.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Created for task: init@0.2.4
+# Creation time: 2025-11-24T00:00:00Z
+
+declare -r pipeline_file=${1:?missing pipeline file}
+
+# 1. Check for init task
+# If init task does not exist, exit 0
+if ! yq -e '.spec.tasks[] | select(.name == "init")' "$pipeline_file" >/dev/null 2>&1; then
```

Comment:
```suggestion
if ! yq -e '(.spec.tasks[], .spec.pipelineSpec.tasks[]) | select(.name == "init")' "$pipeline_file" >/dev/null 2>&1; then
```


By using `pmt-modify`, the pipeline file passed in can either a Pipeline or PipelineRun, it depends on whether users extract the build pipeline definition into a separate YAML file.

> It accepts a single argument, which is a file path pointing to a Pipeline/PipelineRun file including the task bundle update.

Excerpted from [Create a migration](https://github.com/konflux-ci/build-definitions?tab=readme-ov-file#create-a-migration)